src/world_interface/real_world_interface.cpp

```diff
@@ -84,8 +84,12 @@ void initMotors() {
 
 		can::motor::initEncoder(serial, enc_params.isInverted, true, enc_params.ppjr,
 								TELEM_PERIOD);
-		can::motor::setLimitSwitchLimits(serial, enc_params.limitSwitchLow,
-										 enc_params.limitSwitchHigh);
+		auto mDegToPulses = [&](int32_t mdeg) {
+			return static_cast<int32_t>((double)mdeg / 360000.0 * enc_params.ppjr);
+		};
+		// inverted because of weirdness
+		can::motor::setLimitSwitchLimits(serial, mDegToPulses(-enc_params.limitSwitchLow),
+										 mDegToPulses(-enc_params.limitSwitchHigh));
```

Comment:
This is due to some quaddec inversion weirdness in firmware that I don't fully understand. Ask Casey and/or Michael. This should be fixed in firmware and we shouldn't have to negate the parameters.